python/sdist/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=40.6.3", "wheel", "numpy"]
```

Comment:
shouldn't this allow us to remove a couple direct numpy installations in the CI workflow?

Reply:
For setuptools/wheel yes, those I removed. Numpy was so far installed from within `setup`, which worked unreliably and is probably not so great practice. But I found two cases from ancient times - now removed.